packages/react-examples/.storybook/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+    "jsx": "preserve",
+    "noEmit": true,
+    "allowJs": true,
+    "checkJs": true,
+    "allowSyntheticDefaultImports": true,
+    "baseUrl": "../../..",
```

Comment:
keeping this to be consistent with root tsconfig.base.json full paths 
-> packages/font-icons-mdl2/src/index.ts

instead of 

-> ../../font-icons-mdl2/src/index.ts